### PR TITLE
Details functions config

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Create functions to ext
- Create tabs component
- Small fixes

**Related issue(s)**

Part of #1735

### CODE REVIEW

File: examples/resources/serverless/functions.yaml [hunk 1]
```diff
@@ -1,62 +1,221 @@
...
-              "source": "status.conditions[?(@.type=='BuildReady')].status"
-            },
+              "name": "tabs.configuration",
+              "children": [ 
+                {
+                  "widget": "ResourceList",
+                  "source": "$apiRules()",
+                  "name": "configuration.apiRules"
+                }, {
+                  "widget": "ResourceList",
```
Comment (akucharska) [inline examples/resources/serverless/functions.yaml:151]: After the creation of the Subscription from the Function details view, the user is redirected to subscription details. Instead, it should stay on the function detail page.

File: docs/extensibility/display-section.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
 - [Block widgets](#block-widgets)
   - [CodeViewer](#codeviewer)
   - [Columns](#columns)
+  - [Tabs](#tabs)
```
Comment (akucharska) [inline docs/extensibility/display-section.md:18]: Lets keep alphabetical order here

File: docs/extensibility/display-section.md [hunk 3]
```diff
@@ -462,6 +464,37 @@ Columns widgets render the child widgets in multiple columns.
 
 <img src="./assets/display-widgets/Columns.png" alt="Example of a columns widget" style="border: 1px solid #D2D5D9">
 
+### Tabs
```
Comment (akucharska) [inline docs/extensibility/display-section.md:467]: Move it to the end of the section (after `Table`) to keep alphabetical order.

**Review by akucharska — CHANGES_REQUESTED** [review]

File: examples/resources/serverless/functions.yaml [hunk 4]
```diff
@@ -1,62 +1,221 @@
...
+                        "name": "$exists(data.configMapKeyRef) ? data.configMapKeyRef.name : data.secretKeyRef.name",
+                        "namespace": "root.metadata.namespace",
+                        "kind": "$exists(data.configMapKeyRef)
+                        ? 'ConfigMap' 
+                        : 'Secret'"
+                      }
+                    }, {
+                      "name": "code.env.controlledBy",
+                      "widget": "ControlledBy",
+                      "source": "$item.owners"
```
[the remark this answers is not in the document]
Comment (dariadomagala) [reply examples/resources/serverless/functions.yaml:117]: no, it was taken from the owners https://github.com/kyma-project/busola/blob/main/core-ui/src/components/Functions/FunctionDetails/Tabs/Code/FunctionVariables/FunctionVariables.js#L142

[the remark this answers is not in the document]
Comment (akucharska) [reply examples/resources/serverless/functions.yaml:117]: Yes but the `variable` is not part of the function. It is prepared from `customVariables` and `customValueFromVariables`:
```
const entries = [
    ...validateVariables(customVariables),
    ...validateVariables(customValueFromVariables),
  ];
```
those are prepared from `secrets`, `configmaps`, and `function.spec.env` data:
```
const { customVariables, customValueFromVariables } = serializeVariables({
    functionVariables: func?.spec?.env,
    secrets: secrets,
    configmaps: configmaps,
  });
```

Here is the function that prepares variables and the second which takes ownerReferences
https://github.com/kyma-project/busola/blob/main/core-ui/src/components/Functions/helpers/functionVariables/serializeVariables.js#L4
https://github.com/kyma-project/busola/blob/main/core-ui/src/components/Functions/helpers/functionVariables/serializeVariables.js#L65

[the remark this answers is not in the document]
Comment (dariadomagala) [reply examples/resources/serverless/functions.yaml]: We've decided to remove this field

File: docs/extensibility/display-section.md [hunk 5]
```diff
@@ -357,7 +358,7 @@ ResourceLink widgets render internal links to Kubernetes resources.
 #### Widget-specific parameters
 
 - **resource** - To create a hyperlink, Busola needs the name and the kind of the target resource; they must be passed into the **resource** object as property paths in either **data** - value extracted using **source**, or **root** - the original resource. If the target resource is in a `namespace`, provide **namespace**, **name**, and **kind** properties.
-- **linkText** - this property has access to **data** and **root**. This makes it possible to insert resource properties into a translation.
+- **linkText** - a jsonata expression resolving the link text, this property has access to **data** and **root**. To insert dynamic parts of translations, use double quotes `Go to {{data.name}}`.
```
Comment (grego952) [inline docs/extensibility/display-section.md:361]:
```suggestion
- **linkText** - a JSONata expression resolving a link text, this property has access to **data** and **root**. To insert dynamic parts of translations, use double quotes `Go to {{data.name}}`.
```

File: docs/extensibility/display-section.md [hunk 6]
```diff
@@ -416,7 +418,7 @@ CodeViewer widgets display values using a read-only code editor.
 
 #### Widget-specific parameters
 
-- **language** - used for code highlighting. The editor supports languages handled by [Monaco](https://code.visualstudio.com/docs/languages/overview).
+- **language** - a jsonata expression resolving the desired language, used for code highlighting. It has access to the `$root` variable, containing the entire resource. The editor supports languages handled by [Monaco](https://code.visualstudio.com/docs/languages/overview).
```
Comment (grego952) [inline docs/extensibility/display-section.md:421]:
```suggestion
- **language** - a JSONata expression resolving the desired language, used for code highlighting. It has access to the `$root` variable, containing the entire resource. The editor supports languages handled by [Monaco](https://code.visualstudio.com/docs/languages/overview).
```

**Review by grego952 — DISMISSED** [review]

**Review by akucharska — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**